content/departments/engineering/design/goals/index.md

```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
+
+Our team-level goals cascade from the overall company [product vision and
+strategy](../../index.md#product-vision-and-strategy), and each member of the design team
+uses these to define their own individual goals.
+
+## (Draft) Design team goals for the next 3-6 months
+
+### Urgent and important
+
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the scale
+     audit.
+   - Key Result: Design as a function contributes to closing a deal for a
+     2XL customer.
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least `${x}` hours of
```

Comment:
We need to determine what ${x} is. I suggest we use the goal of 1 hour per week (about 2 sessions), and should compensate for the holidays. 

We also need to determine what 'user time' means. I suggest:

> Generative or evaluative research sessions with both internal and external users.

Reply:
I also believe at least 1 hour per week is reasonable—perhaps our intent is "per _typical_ week" (which can be defined with an asterisk)?

For "user time," I'd suggest starting with both internal and external users, but in the following quarter, evolving to prefer external users.

How do we consider things like async usertesting.com sessions? Are they part of this time?

Reply:
I agree on moving primarily toward external. I thnk that user testing session reviews should count toward this time. That said, we'll want to make sure this isn't the only type of testing conducted.

Reply:
Does time spent synthesizing and tagging this information in Dovetail also part of this time? I think it's important to mention/acknowledge that it takes just as much time to pull out the insights from user interviews into Dovetail properly (I'm already guilty of being behind on this myself 😅 ). I'm not necessarily advocating for fewer interviews, but if we mean 1 hour a week _conducting_ user interviews, realistically that's another half hour to another hour tagging and pulling out insights in Dovetail.

Reply:
No, I think this is a measure of time spend with users. I think synthesizing is something we'll get more efficient at as we go.